portal/templates/assigns/assign_manage.html

```diff
@@ -23,7 +23,7 @@ <h3>Assignments for {{ session['session_name'] }}</h3>
 
       <li>
 
-        <h4>{{ assignment['assign_name'] }}</h4>
+        <h4><a href="{{url_for('teacher_views.assign_grade', assign_id=assignment['id'], sessions_id=assignment['sessions_id'], course_id=session['course_id'])}}">{{ assignment['assign_name'] }}</a></h4>
```

Comment:
So we most likely do not want to have the link as part of the assignment name, we want it to have it's own text so it is more user friendly.